feature-libs/organization/administration/components/shared/sub-list/assign-cell.component.ts

```diff
@@ -30,7 +31,7 @@ import { SubListService } from './sub-list.service';
   `,
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class AssignCellComponent<T> extends CellComponent {
+export class AssignCellComponent<T extends BaseItem> extends CellComponent {
```

Comment:
Adding a type to the generic type looks like a potential (compile time) breaking change.

Same for all the other type changes in this PR